src/analytics/constants.ts

```diff
@@ -0,0 +1 @@
+export const ANONYMOUS_USER_ID = 'anonymous_uniswap_user'
```

Comment:
Since we will be using this to set the device_id, should we call it `ANONYMOUS_DEVICE_ID`?

Reply:
👍🏻